go.mod

```diff
@@ -1,9 +1,8 @@
 module google.golang.org/appengine
 
-go 1.11
+go 1.21
```

Comment:
This seems to conflict with the [App Engine support schedule](https://cloud.google.com/appengine/docs/standard/lifecycle/support-schedule#go).